### PR TITLE
[DPS-3135] Extract pushes from Agent & Customer RTM API

### PR DESCRIPTION
**Pull request by oliwiapolec**
# ❗️ Feature branch / Deploy Preview

We suggest to name your branch with `feature/`, for example, `feature/DPS-2947`. See below notifications from **Netlify** about the url.

# 🚀 Links

Replace the placeholder data in links 👇

- [Jira](https://livechatinc.atlassian.net/browse/DPS-3135)

# 📓 Description

Moved pushes to a separate document called "RTM pushes" from all versions of the RTM API. Removed the **Specifics** tables from each push.

### CODE REVIEW

File: src/pages/messaging/customer-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (ale-kacperczyk) [inline src/pages/messaging/customer-chat-api/changelog/index.mdx:1]: As far as I've noticed, a couple of links in the Customer API Changelog lead to `rtm-reference/`, for example **incoming greeting** https://deploy-preview-1035--livechat-public-docs.netlify.app/docs/messaging/customer-chat-api/changelog#greetings

**Review by ale-kacperczyk — COMMENTED** [review]
Besides a couple of links, everything looks good to me.

File: src/pages/messaging/customer-chat-api/changelog/index.mdx (hunk 1 above)
Comment (oliwiapolec) [reply src/pages/messaging/customer-chat-api/changelog/index.mdx:1]: I have no idea how you found those, they seem to be the only ones left to change 😆 should be all good now

**Comment by ale-kacperczyk** [comment]
@bkubiak, obviously, don't go through all the files. Just take a look at how it now works on the preview with the new title, etc.

**Review by ale-kacperczyk — APPROVED** [review]

**Review by bkubiak — APPROVED** [review]
I looked at it, mainly sidebar, navigation, reference from RTM API to pushes and it looks ok 👍🏼  I like the changes 👍🏼

**Pull request merged**